docs/streams/quickstart.html

```diff
@@ -40,10 +40,10 @@
 final Serde&lt;String&gt; stringSerde = Serdes.String();
 final Serde&lt;Long&gt; longSerde = Serdes.Long();
 
-// Construct a `KStream` from the input topic "streams-wordcount-input", where message values
+// Construct a `KStream` from the input topic "streams-plaintext-input", where message values
```

Comment:
It is planned on a follow-up PR. I just created this JIRA to cover the whole story:

> I've verified that it all works, but as i said above i don't see any instructions in the quickstart docs for how to run this. It refers to examples etc and there is no mention of building the mvn project (unless i'm blind)

I'm adding the web docs at the same time while trying to solicit some feedbacks on the project itself now.